runtime/lua/vim/lsp/completion.lua

```diff
@@ -47,6 +47,9 @@ local ns_to_ms = 0.000001
 --- @field triggers table<string, vim.lsp.Client[]>
 --- @field convert? fun(item: lsp.CompletionItem): table
 
+--- @class vim.lsp.completion.trigger.Opts
```

Comment:
This is kind of an awkward name and we don't really want to bikeshed it (though, I know, we don't worry about type names in general), but inlining it seems to make sense for now.

```suggestion
--- @class vim.lsp.completion.trigger.Opts
--- @inlinedoc
```

Reply:
Ehhh I don't have strong feelings about this one. Although I don't have anything concrete in mind I can imagine these options being modified in the future when we add inline completion support, so I prefer the separate definition.

Reply:
`@inlinedoc` only affects docs, or maybe I misunderstood you?

Reply:
Oh gotcha. Let's do that.